backend/src/Controller/Admin/Notification/AdminNotificationToUsersController.php

```diff
@@ -0,0 +1,326 @@
+     *   ),
+     * )
+     *
+     * @Security(name="Bearer")
+     */
+    public function updateAdminNotification(Request $request): JsonResponse
+    {
+        $data = json_decode($request->getContent(), true);
+
+        $request = $this->autoMapping->map(\stdClass::class, AdminNotificationUpdateRequest::class, (object) $data);
+
+        $violations = $this->validator->validate($request);
+
+        if (\count($violations) > 0) {
+            $violationsString = (string) $violations;
+
+            return new JsonResponse($violationsString, Response::HTTP_OK);
+        }
+
+        $result = $this->adminNotificationToUsersService->updateAdminNotification($request);
+
+        if(isset($result->state)) {
+         
+            return $this->response($result, self::NOTIFICATION_NOT_FOUND);
+        } 
+
+        return $this->response($result, self::UPDATE);
+    }
+
+    /**
+     * admin: delete Admin Notification
+     * @Route("adminnotification/{id}", name="deleteAdminNotification", methods={"DELETE"})
+     * @IsGranted("ROLE_ADMIN")
+     * @param Request $request
+     * @return JsonResponse
+     *
+     * @OA\Tag(name="Admin Notification")
+     *
+     * @OA\Parameter(
+     *      name="token",
+     *      in="header",
+     *      description="token to be passed as a header",
+     *      required=true
+     * )
+     *
+     * @OA\Response(
+     *      response=401,
+     *      description="Returns notification",
+     *      @OA\JsonContent(
+     *          @OA\Property(type="string", property="status_code"),
+     *          @OA\Property(type="string", property="msg"),
+     *          @OA\Property(type="object", property="Data",
+     *            @OA\Property(type="integer", property="id"),
+     *            @OA\Property(type="string", property="title"),
+     *            @OA\Property(type="string", property="msg"),
+     *            @OA\Property(type="string", property="appType"),
+     *            @OA\Property(type="integer", property="userId"),
+     *      ),
+     *   ),
+     * )
+     *
+     * or
+     * 
+     * @OA\Response(
+     *      response="default",
+     *      description="Returns notification",
+     *      @OA\JsonContent(
+     *          @OA\Property(type="string", property="status_code", description="9401"),
+     *          @OA\Property(type="string", property="msg", description="notification not exist Successfully."),
+     *          @OA\Property(type="object", property="Data",
+     *            @OA\Property(type="integer", property="state"),
+     *      ),
+     *   ),
+     * )
+     *
+     * @Security(name="Bearer")
+     */
+    public function deleteAdminNotification($id): JsonResponse
+    {
+        $result = $this->adminNotificationToUsersService->deleteAdminNotification($id);
+
+        if(isset($result->state)) {
+         
+            return $this->response($result, self::NOTIFICATION_NOT_FOUND);
+        } 
+
+        return $this->response($result, self::DELETE);
```

Comment:
Here we may find another better situation, like the final return statement we may put it in an else statement.